homeassistant/components/switch/broadlink.py

```diff
@@ -288,6 +289,12 @@ def should_poll(self):
         """Return the polling state."""
         return True
 
+    @property
+    def device_state_attributes(self):
```

Comment:
Add `self._load_power`  as `current_power_w` property instead

Reply:
change property 'load_power' to 'current_power_w' instead

Reply:
ok，got it.